src/Models/Nonhydrostatic2DModels/nonhydrostatic_2d_model.jl

```diff
@@ -140,3 +140,10 @@ function Models.make_ode_function(model::Nonhydrostatic2DModel{FT}) where {FT}
         return dY
     end
 end
+
+function get_velocities(u, model::Nonhydrostatic2DModel)
+    u_1 = getproperty(u.nhm,:ρuh) ./ getproperty(u.nhm, :ρ)
```

Comment:
`u.nhm` is hard coded here. I think `getproperty(u, model.name)` should give `u.nhm` so that the hard coding can be avoided.